Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -641,14 +641,22 @@ CFStringRef _DWriteGetFamilyNameForFontName(CFStringRef fontName) {
                           CFArrayRef fontNames = _DWriteGetFontNamesForFamilyName(familyName);
 
                           for (size_t j = 0; j < CFArrayGetCount(fontNames); j++) {
-                              CFDictionaryAddValue(initMap, static_cast<CFStringRef>(CFArrayGetValueAtIndex(fontNames, j)), familyName);
+                              CFStringRef systemFontName = static_cast<CFStringRef>(CFArrayGetValueAtIndex(fontNames, j));
+                              CFMutableStringRef upperSystemFontName =
+                                  CFStringCreateMutableCopy(nullptr, CFStringGetLength(systemFontName), systemFontName);
+                              CFAutorelease(upperSystemFontName);
+                              CFStringUppercase(upperSystemFontName, CFLocaleCopyCurrent());
```

Comment:
nit: you could've also used CFLocaleGetSystem() but whatever

---
In reply to: [80806159](https://github.com/Microsoft/WinObjC/pull/1042#discussion_r80806159) [](ancestors = 80806159)